qlty-config/src/config/plugin.rs

```diff
@@ -715,7 +715,15 @@ pub enum IssueMode {
     Disabled = 4,
 }
 
-impl IssueMode {
+impl IssueMode {pub fn to_string(&self) -> String {
```

Comment:
This formatting is wrong. Rust has a canonical auto formatted which you should be able to get your editor to run on save.